CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
 ### [Versions](https://mui.com/versions/)
 
+## 5.1.1
+
+<!-- generated comparing v5.1.0..master -->
+
+_Nov 15, 2021_
+
+A big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- 🛠 Rename `@mui/core` to `@mui/base`. (#29585) @michaldudak
+- And many more 🐛 bug fixes and 📚 improvements.
+
+### `@mui/material@5.1.1`
+
+- &#8203;<!-- 34 -->[Breadcrumbs][divider] Replace decimal spacing values with integers and css calc (#29526) @anikcreative
+- &#8203;<!-- 10 -->[Select][nativeselect] Add `multiple` class (#29566) @aaronholla
+- &#8203;<!-- 09 -->[Popper] Split into PopperUnstyled and Popper (#29488) @michaldudak
+- &#8203;<!-- 08 -->[Select] Make it clear that `Select` is not a root component (#29593) @hbjORbj
+- &#8203;<!-- 13 -->[l10n] Improved Dutch (nl-NL) locale (#29592) @flipvrijn
+
```

Comment:
Do we miss a formal entry for `@mui/base`? To document the breaking change.

Reply:
Agree, it is a breaking change we should state it in a new entry under the new package name 👍

Reply:
Done.